src/core/layout/qgslayoutitemattributetable.h

```diff
@@ -357,6 +357,7 @@ class CORE_EXPORT QgsLayoutItemAttributeTable: public QgsLayoutTable
 
     QList< QList< QgsConditionalStyle > > mConditionalStyles;
     QList< QgsFeature > mFeatures;
+    QMap<QString, QVariant> mLayerCache;
```

Comment:
I can't see where this is ever cleared? Probably it should be done at the start of getTableContents?

Reply:
Yes, done.